scripts/scil_connectivity_compute_simple_matrix.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Computes a very simple connectivity matrix, using the streamline count and the
+position of the streamlines' endpoints.
+
+This script is intented for exploration of your data. For a more thorough
+computation (using the longest streamline segment), and for more options about
+the weights of the matrix, see:
+>> scil_connectivity_compute_matrices.py
+
+Contrary to scil_connectivity_compute_matrices.py, works with an incomplete
+parcellation (i.e. with streamlines ending in the background).
+
+In the output figure, 4 matrices are shown, all using the streamline count:
+    - Raw count
+    - Raw count (log view)
+    - Binary matrix (if at least 1 streamline connects the two regions)
+    - Percentage of the total streamline count.
+
+You may select which matrix to save to disk (as .npy) using options --binary or
+--percentage. Default ouput matrix is the raw count.
+"""
+
+import argparse
+import logging
+import os.path
+
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import nibabel as nib
+import numpy as np
+
+from scilpy.connectivity.connectivity import \
+    compute_triu_connectivity_from_labels
+from scilpy.image.labels import get_data_as_labels
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import assert_inputs_exist, assert_outputs_exist, \
+    add_verbose_arg, add_overwrite_arg, assert_headers_compatible, \
+    add_reference_arg
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_labels',
+                   help='Input nifti volume.')
+    p.add_argument('streamlines',
+                   help='Tractogram (trk or tck).')
+    p.add_argument('out_matrix',
+                   help="Out .npy file.")
+    p.add_argument('out_labels',
+                   help="Out .txt file. Will show the ordered labels (i.e. "
+                        "the columns and lines' tags).")
+    p.add_argument('--hide_background', nargs='?', const=0, type=int,
```

Comment:
This should be on by default, and I believe it should be removed from the matrix altogether.
(instead of masked)

--keep background with a store_true maybe?
To replace --hide_background I would add a --mask_labels (with an nargs='+' so people can mask the row/column they want)